lightning/mydump/parser.go

```diff
@@ -371,6 +371,9 @@ func (parser *ChunkParser) ReadRow() error {
 	for {
 		tok, content, err := parser.lex()
 		if err != nil {
+			if err == io.EOF && st != stateValues {
+				return errors.Errorf("Syntax error: premature EOF at offset %d", parser.pos)
```

Comment:
I think the error message starting with lower-case is more appropriate. See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings